### ISSUE TITLE
[Topic] Feedback on form errors

### ISSUE THREAD

**Issue by rianrietveld** [wpaccessibility/wp-a11y-docs#161]
URL: https://wpaccessibility.org/docs/topics/forms/feedback/
Code: https://github.com/wpaccessibility/wp-a11y-docs/blob/main/docs/topics/forms/feedback.md

Please read [How to contribute](https://wpaccessibility.org/docs/contribute/) before you start.

## Tasks

This content needs to be rewritten and expanded.

See for the structure of an accessibility topic: 
Write documentation: [Content structure of an accessibility topic](https://wpaccessibility.org/docs/contribute/write/#content-structure-of-an-accessibility-topic)

### Include pros and cons about HTML5 client side form validation

The topic Mark required for screen readers referes to it and theis topic belongs here, I propose a dedicated page.

Draft text:
> Using `required` triggers the [client-side form validation](https://developer.mozilla.org/en-US/docs/Learn_web_development/Extensions/Forms/Form_validation) of the browser. 
At the time of writing, this validation may have accessibility issues. The error messages are not specific enough, they disappear after a few seconds, and they currently lack sufficient screen reader support.
If you decide to use `required`, also use `novalidate` on the form element to prevent the browser's form validation from kicking in and enable you to handle validation server-side.

Also add: transation issues and browser inconsistancy
